WhaleSpotting/Controllers/UserController.cs

```diff
@@ -43,12 +43,12 @@ public IActionResult Create([FromBody] UserRequest newUser)
         return Created(url, new UserResponse(user));
     }
 
-    [HttpGet("")]
-    public ActionResult<List<UserResponse>> ListOfUsers()
+    [HttpGet("leaderboard")]
+    public ActionResult<List<UserLeaderboardResponse>> ListOfUserLeaderboard()
```

Comment:
Can we rename this endpoint to `getUserLeaderboard`?
```suggestion
    public ActionResult<List<UserLeaderboardResponse>> GetUserLeaderboard()
```